metadata.yml

```diff
@@ -1,3 +1,8 @@
 name: "keystone"
 version: "2016.4.1"
-source: "https://github.com/openstack/salt-formula-keystone"
+source: "https://github.com/salt-formulas/salt-formula-keystone"
+dependencies:
```

Comment:
I know it's not documented yet (my fault, I will punish myself and go sleep without dinner) but you also need to update dependencies in `debian/control` file which lives in `debian/unstable` branch to avoid breaking package build.